patroni/dcs/__init__.py

```diff
@@ -402,21 +534,37 @@ def from_node(version: _Version, value: Union[str, Dict[str, str]]) -> 'Failover
         return Failover(version, data.get('leader'), data.get('member'), data.get('scheduled_at'))
 
     def __len__(self) -> int:
+        """Implement ``len`` function capability."""
```

Comment:
```suggestion
        """Implement ``len`` function capability.
        
        .. note::
           This magic method aids in the evaluation of "emptiness" of a ``Failover`` instance. For example:
           
           >>> failover = Failover.from_node(1, None)
           >>> len(failover)
           0
           >>> assert bool(failover) is False
           
           >>> failover = Failover.from_node(1, {"leader": "cluster_leader"})
           >>> len(failover)
           1
           >>> assert bool(failover) is True
        
           This makes it easier to write ``if cluster.failover`` rather than the longer statement.
        
        """
```

Reply:
Looks good